pages/docs/balance/bank-funding-source.mdx

```diff
@@ -33,13 +33,13 @@ Dwolla supports traditional `checking` and `savings` accounts to be added and ve
-| [Dwolla + Secure Exchange solution](https://developers.dwolla.com/api-reference/exchanges)                   | Yes                                                     | Online banking credentials                 | ~85%              | Checking, <br /> Savings                                     | [Tokenized integration](https://developers.dwolla.com/concepts/secure-exchange)                                                                                                   |
-| Other Approved Third-party Provider                                                                          | Yes, if part of third party offering                    | Variable                                   | Variable          | Variable                                                     | Variable                                                                                                                                                                          |
+| Bank Addition Method                                                                                         | Automatic Verified Status (i.e. Eligible to Send Funds) | Information Required                       | U.S Bank Coverage | Supported Bank Account Types                                 | Other features                                                                                                                                             |
+| ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------- | ------------------------------------------ | ----------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Dwolla API](https://developers.dwolla.com/api-reference/funding-sources/create-funding-source-for-customer) | No                                                      | Bank Account Number, <br /> Routing number | 100%              | Checking, <br /> Savings, <br /> Loan, <br /> General Ledger | Optional bank verification with [Microdeposits](https://developers.dwolla.com/guides/micro-deposit-verification#verify-a-bank-account-with-micro-deposits) |
+| [Drop-in components](https://developers.dwolla.com/docs/drop-in-components)                                  | No                                                      | Bank Account Number, <br /> Routing number | 100%              | Checking, <br /> Savings                                     | Tokenized data, <br /> Optional bank verification with [Microdeposits](https://developers.dwolla.com/docs/drop-in-components#verify-micro-deposits)        |
```

Comment:
#create-a-funding-source

Reply:
Addressed in this [commit](https://github.com/Dwolla/dev-portal/pull/584/commits/12439e8f930f1bcb1a70014e2243a40946880005).